docs/developer-docs/latest/setup-deployment-guides/configurations/optional/rbac.md

```diff
@@ -1,10 +1,8 @@
 ---
 title: Role-Based Access Control (RBAC) - Strapi Developer Documentation
-description: 
+description: In Strapi, Role-Based Access Control (RBAC) is an approach to restricting access to some features of the admin panel to some users. The Community Edition of Strapi offers 3 default roles. To go further, creating custom conditions for any type of permission is also possible and requires an Enterprise Edition.
```

Comment:
This description is too long, it would be nice to not exceed 160 characters :)

Reply:
```suggestion
In Strapi, RBAC is an approach to restricting access to some features of the admin panel to some users. The Community Edition of Strapi offers 3 default roles.
```